arcgis-ios-sdk-samples/Layers/Apply unique values with alternate symbols/ApplyUniqueValuesAlternateSymbolsViewController.swift

```diff
@@ -0,0 +1,113 @@
+// Copyright © 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class ApplyUniqueValuesAlternateSymbolsViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            // Set the map and its initial viewpoint.
+            mapView.map = AGSMap(basemapStyle: .arcGISTopographic)
+            mapView.map?.operationalLayers.add(featureLayer)
+            mapView.setViewpoint(viewpoint)
+            // Add a handler to update the current label.
+            mapView.viewpointChangedHandler = { [weak self] in
+                DispatchQueue.main.async {
+                    self?.changeScaleLabel()
+                }
+            }
+        }
+    }
+    
+    @IBOutlet var currentScaleLabel: UILabel!
+    @IBOutlet var resetViewpointBarButtonItem: UIBarButtonItem!
+    
+    /// Response to the bar button item being tapped.
+    @IBAction func resetViewpointTapped(_ button: UIBarButtonItem) {
+        // Set the viewpoint with animation.
+        mapView.setViewpoint(viewpoint, duration: 5, curve: .easeInOutSine)
+    }
+    
+    /// The feature service URL.
+    static let featureServiceURL = URL(string: String("https://sampleserver6.arcgisonline.com/arcgis/rest/services/SF311/FeatureServer/0"))!
+    /// The feature layer set in San Francisco, CA.
+    let featureLayer = AGSFeatureLayer(featureTable: AGSServiceFeatureTable(url: featureServiceURL))
+    let viewpoint = AGSViewpoint(center: AGSPoint(x: -13632095.660131, y: 4545009.846004, spatialReference: .webMercator()), scale: 7_500)
+    
+    /// The formatter used to generate strings from scale values.
+    private let scaleFormatter: NumberFormatter = {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .decimal
+        numberFormatter.maximumFractionDigits = 0
+        return numberFormatter
+    }()
+    
+    /// Creates the unique values renderer for the feature layer.
+    func createUniqueValuesRenderer() {
```

Comment:
I think it's better to change it to `func makeUniqueValuesRenderer() -> AGSUniqueValueRenderer`, and set the feature layer's renderer before it's added to the map's operational layers.